### PR TITLE
E2E Tests: Try using a shared package for some of the utilities

### PR DESCRIPTION
**Pull request by bsessions85**
#### Changes proposed in this Pull Request:

* This PR pulls out some of the e2e test functionality that can be shared and pulls in the new `puppeteer-utils` package instead.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* If you're an Automattician, include a shortlink to the p2 discussion with Jetpack Product here.
pbsmUs-d-p2

#### Testing instructions:

* Ensure that the e2e tests pass and run locally and in Travis
*

#### Proposed changelog entry for your changes:

* no changelog entry needed

### CODE REVIEW

File: tests/e2e/lib/setup.js [hunk 1]
```diff
@@ -1,27 +1,12 @@
 /**
  * External dependencies
  */
-import config from 'config';
+const config = require( 'config' );
```
[the remark this answers is not in the document]
Comment (bsessions85) [reply tests/e2e/lib/setup.js]: No, good catch. Updated it.

**Comment by bsessions85** [comment]
@brbrr  This PR should be ready to go when you're ready. I rebased and pulled in the latest `puppeteer-utils`.

**Comment by kraftbj** [comment]
@brbrr I'm marking this for author reply since it ended up going stale since the last rebase. Do you have the bandwidth to get it ready for review?

**Comment by brbrr** [comment]
Closing this as it is not relevant anymore

**Pull request closed without merging**